packages/server/src/server.js

```diff
@@ -80,7 +75,6 @@ const grantConfig = {
 mongoose
   .connect(DATABASE_URL, {
     useNewUrlParser: true,
-    dbName: DATABASE_NAME,
     user: DATABASE_USER,
     pass: DATABASE_PASSWORD,
   })
```

Comment:
It's slightly easier setup to have the dbName as part of the database URL when copying from the MongoDB Atlas.